generator/main.ts

```diff
@@ -69,8 +69,11 @@ async function main([out_dir, ...idl_files]) {
     const parsedData = webidl.parse(await file.read(path.resolve(idl_file)));
     const idl_fragments: idls.Fragments = new idls.Fragments(parsedData);
     await generateInterface(env, idl_file, out_dir, idl_fragments);
-    if (idl_fragments.definitions[0] instanceof idls.InterfaceImpl)
-      interface_names.push(new String(idl_fragments.definitions[0].name));
+    for (const definition of idl_fragments.definitions) {
+      if (definition instanceof idls.InterfaceImpl) {
```

Comment:
After remove Enum in DefinitionTypes, we can remove instanceof keyword.
So, I think recommend add "FIXME" for remember next.